h2o-py/requirements.txt

```diff
@@ -2,6 +2,8 @@ pip
 colorama>=0.3.8
 future
 requests
+winkerberos >= 0.5.0; sys.platform == 'win32'
```

Comment:
Just please keep in mind It shouldn't be a hard requirement, only an optional one.